mitodl/ocw-hugo-themes
# fix: placing mp_logo in static images of course theme

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/289

#### What's this PR do?
Creates directory of static and images (inside static) in course theme and places the logo inside it so it can be rendered in any course.

#### How should this be manually tested?
- Go to any course which uses this logo and verify that logo image is showing instead of alt text "Buy at MIT Press". (course [21g.049-spring-2017](https://github.mit.edu/ocw-content-rc/21g.049-spring-2017) use this logo in pages/assignments/presentations-and-sources/ under section of week 5)

#### Screenshots (if appropriate)
Before:
![image](https://user-images.githubusercontent.com/93309234/149085129-3bc0d2ca-11a8-455e-ad2b-be0c073f8dd4.png)

After: 
![image](https://user-images.githubusercontent.com/93309234/149085048-2feb0432-e825-4430-8429-87d1f2361749.png)
